docs/sources/helm-charts/mimir-distributed/configure/configure-autoscaling.md

```diff
@@ -0,0 +1,194 @@
+---
+description: Learn how to configure Grafana Mimir autoscaling when using Helm.
+menuTitle: Configure autoscaling
+title: Configure Grafana Mimir autoscaling with Helm
+weight: 30
+---
+
+# Configure Grafana Mimir autoscaling with Helm
+
+{{< admonition type="warning" >}}
+Autoscaling support in the Helm chart is currently experimental. Use with caution in production environments and thoroughly test in a non-production environment first.
+{{< /admonition >}}
+
+This topic explains how to configure autoscaling for Mimir components using KEDA and Kubernetes Horizontal Pod Autoscaler (HPA).
+
+## Before you begin
+
+- Ensure you have a running Mimir cluster deployed with Helm
+- Verify you have the required permissions to modify Helm deployments
+- Familiarize yourself with Kubernetes HPA concepts
+
+## Prerequisites
+
+To use autoscaling, you need:
+
+1. KEDA installed in your Kubernetes cluster
+2. Prometheus metrics available for scaling decisions
+
+{{< admonition type="warning" >}}
+Do not use the same Mimir or Grafana Enterprise Metrics cluster for storing and querying autoscaling metrics. Using the same cluster can create a dangerous feedback loop:
+
+1. If the Mimir/GEM cluster becomes unavailable, autoscaling will stop working because it cannot query the metrics
+2. This can prevent the cluster from automatically scaling up during high load or recovery
+3. The inability to scale can further exacerbate the cluster's unavailability
+4. This may prevent the cluster from recovering
+
+Instead, use a separate Prometheus instance or a different metrics backend for autoscaling metrics.
```

Comment:
```suggestion
Don't use the same Mimir or Grafana Enterprise Metrics cluster for storing and querying autoscaling metrics. Using the same cluster can create a dangerous feedback loop.

For instance, if the Mimir or GEM cluster becomes unavailable, autoscaling stops working, because it cannot query the metrics. This prevents the cluster from automatically scaling up during high load or recovery. This inability to scale further exacerbates the cluster's unavailability, which might, in turn, prevent the cluster from recovering.

Instead, use a separate Prometheus instance or a different metrics backend for autoscaling metrics.
```

Reply:
You have this formatted as an ordered list, but it's not really a sequence of steps. I think it works better to describe the feedback loop in paragraph form.